### DISCUSSION TITLE
[LoginUI] Set HTML lang attribute to "en" when internationalization disabled

### DISCUSSION THREAD

**Discussion by andreas-blaettlinger** [keycloak/keycloak#35050] [Ideas]
I am currently looking through the LoginUI and noticed the following line in e.g. the template.ftl:

`<html class="${properties.kcHtmlClass!}"<#if realm.internationalizationEnabled> lang="${locale.currentLanguageTag}"</#if>>`

My understanding is that the `lang` attribute is only set when internationalization is enabled and when internationalization is disabled, the `lang` is not set at all. Shouldn't we set it to `en` when internationalization is disabled?

As this line is used in several templates it could be cleaner when the logic is centralized and only a generated property `langAttribute` is used in the template.

Could look like this:

`<html class="${properties.kcHtmlClass!}" lang="${langAttribute}">`

Any ideas on that?

**Comment by jonkoops** [comment]
This sounds like a good proposal, we'd welcome this as a contribution. Can you log an issue for it?
